sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -952,12 +960,12 @@ def download_manifest(self, repository: str, tag_or_digest: str, **kwargs) -> Do
                 self._client.container_registry.get_manifest(
                     name=repository,
                     reference=tag_or_digest,
-                    headers={"Accept": OCI_MANIFEST_MEDIA_TYPE},
+                    headers={"Accept": OCI_IMAGE_MANIFEST},
```

Comment:
We had a conversation with the service team, and decided that we will request all known media in the accept header for any "GET manifest" request.

If you'd like, you can copy what .NET is doing for the accept header [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Content/ContainerRegistryContentClient.cs#L619-L637), it should be up to date.